benchmark/elastic/issue_elastic_queries.py

```diff
@@ -0,0 +1,126 @@
+output_file = OUTPUT_FILE_NAME
+queries = QUERIES
+aggregation_fields = AGGREGATION_FIELDS
+
+def runQuery(query):
+    cmd = query + " > " + output_directory + "query-output.json"
+    os.system(cmd)
+    with open(output_directory + "query-output.json", 'r') as j:
+        contents = json.loads(j.read())
+        executiontime = contents["took"]
+        hits = contents["hits"]["total"]["value"]
+
+    os.system("rm " + output_directory + "query-output.json")
+    return {"executiontime": executiontime * 0.001, "hits": hits}
+
+def getQuery(queryname, arguments):
+    # TODO: edit queries to return all results, not just top 10-1000
+    if queryname == "search id.orig_h":
+        query = "curl -X GET 'http://localhost:9200/test/_search?q=id.orig_h:"+arguments[0]+"&format=json&pretty'"
+    elif queryname == "search id.orig_h + sort ts":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"sort\" : [\"ts\" ],\"query\" : {\"term\" : { \"id.orig_h\" : \""+arguments[0]+"\" }}}'"
+    elif queryname == "search id.orig_h + sort ts + slice 5":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"size\": 5,\"sort\" : [{\"ts\" : {\"order\": \"asc\" }}],\"query\" : {\"term\" : { \"id.orig_h\" : \""+arguments[0]+"\" }}}'"
+    elif queryname == "search id.orig_h + count by id.resp_h":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"query\" : {\"term\" : { \"id.orig_h\" : \""+ arguments[0] +"\" }}, \"aggs\": {\"id.resp_h\": {\"terms\": {\"field\": \"id.resp_h\"}}}}'"
+    elif queryname == "search id.orig_h + sum orig_bytes":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"query\" : {\"term\" : { \"id.orig_h\" : \""+ arguments[0] +"\" }}, \"aggs\": {\"orig_bytes\": {\"sum\": {\"field\": \"orig_bytes\"}}}}'"
+    elif queryname == "search id.orig_h + count by schema":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"query\" : {\"term\" : { \"id.orig_h\" : \""+ arguments[0] +"\" }}, \"aggs\": {\"schema\": {\"terms\": {\"field\": \"_path\"}}}}'"
+    else:
+        query= ""
+    return query
+
+def issueQueries():
+    real_list = []
+    query_name_list = []
+    argument_list = []
+    validation_list =[]
+    
+    reset()
+
+    with open(input_directory + workload_file ) as f:
+        data = ndjson.load(f)
+        for d in data:
+            if (d['query'] in queries):
+                queryname = d['query']
+                query = getQuery(queryname, d['arguments'])
+                if query:
+                    if queryname in aggregation_fields:
+                        prepForAggregation(aggregation_fields[queryname])
+
+                    queryoutput = runQuery(query)
```

Comment:
Instead of adding all of these fields to different arrays and then writing them out once you're done issuing queries, it might be simpler to write them out to a file as you go along. There's an example of that in `benchmark/zed/z_benchmark_sequence.py` (see [here](https://github.com/aousterh/data_model/blob/284c56e85a7251360841923e026481368fc98fb9/benchmark/zed/z_benchmark_sequence.py#L70)). Or you could use the approach in `benchmark/zed/mongo/benchmark.py` where you have a single array for all results and write them out at the end (see [here](https://github.com/aousterh/data_model/blob/284c56e85a7251360841923e026481368fc98fb9/benchmark/mongo/benchmark.py#L82)). Basically, I'm suggesting to simplify the code a bit so that if we want to add another output field later, we don't have to change the code in several different places.